### DISCUSSION TITLE
请问大佬有用jest测试过antv图表吗

### DISCUSSION THREAD

**Discussion by Cherry860** [haixiangyan/jest-tutorial#39] [General]
测试引用了antv柱状图的组件报了这个错误，搜了好多资料都没看到过这方面的实践，这俩结合目前能结合使用吗
![image](https://user-images.githubusercontent.com/37651304/187400447-22593bb7-b22f-41d0-ab14-6c0b7b6f85a8.png)

**Comment by haixiangyan** [comment]
Column 里可能用到一些子组件（类似 Worker？），是不是用了 shallowMount 的问题？
